File: CS50x/pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //created variable 
    int blocks = 0;

    //loop around atleast once
    do
    {
        blocks = get_int("Height between 1 and 8: ");

        if (blocks < 1 || blocks > 8)
        {
            continue;
        }

        for (int i = 0; i < blocks; ++i)
        {
            for (int j = 0; j < blocks; ++j)
            {
                /*add hash when block - i-1*/
                if (j >= (blocks - (i + 1)))
                {
                    printf("#");
                }
                else
                {
                    printf(" ");
                }
            }
            
            /*adding new line every time the j loop
             *finishes
             */
            printf("\n");
        }

    } 
    while (blocks < 1 || blocks > 8);
}

File: CS50x/pset6/mario/less/mario.py
from cs50 import get_int

block = 0

while True:
    # get user input
    block = get_int("Height: ")

    # check if block is not withint range
    if block < 1 or block > 8:
        continue

    # loop around and print the blocks
    for i in range(block):
        for j in range(block):
            # make sure to only print necesary blocks
            if j >= (block - (i + 1)):
                print("#", end="")
            else:
                print(" ", end="")
        # print new line
        print()
        
    # break from while loop
    if block in range(9):
        break